src/main.rs

```diff
@@ -23,7 +23,7 @@ use std::{
 use tokio::task::spawn_blocking;
 use verify::VerificationAnnotations;
 
-use tracing::{debug, info, warn};
+use tracing::{debug, error, info, warn};
```

Comment:
(randomly opening thread here)

We should also update the help info on all the definitions of `docker-config-json-path` flags.
E.g:
https://github.com/kubewarden/kwctl/blob/42de8bab9e6aedef1abbb8d3e0e3147c61276d64/src/cli.rs#L24

Reply:
Done